modules/afsmtp/afsmtp.c

```diff
@@ -45,6 +45,7 @@ typedef struct
 {
   gchar *phrase;
   gchar *address;
+  LogTemplate *addr_tmpl;
   afsmtp_rcpt_type_t type;
```

Comment:
If we are not using the string representation  I would get rid off those everywhere.

Also, LogTemplate contains the template string in addition to the compiled representation.